npm/src/language-service/language-service.ts

```diff
@@ -115,13 +119,17 @@ export class QSharpLanguageService implements ILanguageService {
     return result;
   }
 
-  async getHover(documentUri: string, offset: number): Promise<IHover | null> {
+  async getHover(
+    documentUri: string,
+    offset: number
+  ): Promise<IHover | undefined> {
     const code = this.code[documentUri];
+    if (!code) {
```

Comment:
Same as above. I expect it's valid for a document to be empty (e.g. a new file, or just deleted all the contents to start again).

Reply:
Ooh, good catch. Thank you.